Unable to build

### Checklist

- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I will participate in further discussion about this issue and can help by testing (if requested).

### Card Version

master

### Component Version

_No response_

### Bug description

Running "npm install" from a clean checkout gives errors because of package version conflicts

```
$ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @rollup/plugin-commonjs@15.1.0
npm ERR! Found: rollup@1.32.1
npm ERR! node_modules/rollup
npm ERR!   rollup@"^1.32.1" from the root project
npm ERR!   peer rollup@"^1.20.0 || ^2.0.0" from @rollup/plugin-json@4.1.0
npm ERR!   node_modules/@rollup/plugin-json
npm ERR!     @rollup/plugin-json@"^4.0.0" from the root project
npm ERR!   6 more (@rollup/pluginutils, rollup-plugin-babel, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer rollup@"^2.22.0" from @rollup/plugin-commonjs@15.1.0
npm ERR! node_modules/@rollup/plugin-commonjs
npm ERR!   @rollup/plugin-commonjs@"^15.0.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: rollup@2.79.1
npm ERR! node_modules/rollup
npm ERR!   peer rollup@"^2.22.0" from @rollup/plugin-commonjs@15.1.0
npm ERR!   node_modules/@rollup/plugin-commonjs
npm ERR!     @rollup/plugin-commonjs@"^15.0.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Steps to reproduce

Clone a fresh copy of the repo and run "npm install"

### Additional info

_No response_

--- a/package.json
+++ b/package.json
@@ -27,8 +27,6 @@
     "@babel/plugin-proposal-decorators": "^7.4.0",
     "@formatjs/intl-utils": "^3.8.4",
     "@mdi/js": "^6.4.95",
-    "@rollup/plugin-commonjs": "^15.0.0",
-    "@rollup/plugin-json": "^4.0.0",
     "@typescript-eslint/eslint-plugin": "^2.6.0",
     "@typescript-eslint/parser": "^2.6.0",
     "custom-card-helpers": "1.8.0",
@@ -41,6 +39,8 @@
     "lit": "^2.0.0",
     "rollup": "^1.32.1",
     "rollup-plugin-babel": "^4.3.3",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-json": "^4.0.0",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-serve": "^1.0.1",
     "rollup-plugin-terser": "^5.1.2",
@@ -52,6 +52,6 @@
     "babel-cli": "^6.26.0",
     "eslint": "^6.8.0",
     "prettier": "1.19.1",
-    "rollup-plugin-visualizer": "^5.5.2"
+    "rollup-plugin-visualizer": "^4.2.0"
   }
 }

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,10 +1,10 @@
 import nodeResolve from 'rollup-plugin-node-resolve';
 import typescript from 'rollup-plugin-typescript2';
 import babel from 'rollup-plugin-babel';
-import json from '@rollup/plugin-json';
+import json from 'rollup-plugin-json';
 import { terser } from 'rollup-plugin-terser';
-import commonjs from '@rollup/plugin-commonjs';
-import { visualizer } from 'rollup-plugin-visualizer';
+import commonjs from 'rollup-plugin-commonjs';
+import visualizer from 'rollup-plugin-visualizer';
 
 
 const plugins = [

